app/view_models/auction_show_view_model.rb

```diff
@@ -73,7 +73,9 @@ def bid_status_class
   end
 
   def bid_status_partial
-    if auction.type == 'reverse' && (over? || available_and_user_is_winning_bidder?)
+    if reverse_auction_over? ||
```

Comment:
this is related to the bid status for a user rather than the rules of the auction -- but do you mean we should create rules-type objects?